dbft_test.go

```diff
@@ -0,0 +1,96 @@
+package dbft
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/CityOfZion/neo-go/pkg/util"
+	"github.com/nspcc-dev/dbft/block"
+	"github.com/nspcc-dev/dbft/crypto"
+	"github.com/nspcc-dev/dbft/payload"
+)
+
+type Payload = payload.ConsensusPayload
+
+type testState struct {
+	myIndex    int
+	count      int
+	privs      []crypto.PrivateKey
+	pubs       []crypto.PublicKey
+	ch         chan Payload
+	currHeight uint32
+	currHash   util.Uint256
+}
+
+func TestDBFT_OnStartPrimarySendPrepareRequest(t *testing.T) {
+	const validatorsCount = 7
+
+	s := newTestState(2, validatorsCount)
+
+	t.Run("backup sends nothing on start", func(t *testing.T) {
```

Comment:
really like when tests look like this. We need to use the same approach in neo-go